src/test/java/uk/gov/hmcts/reform/sscs/service/SendNotificationServiceTest.java

```diff
@@ -518,6 +605,38 @@ private void verifyNotificationIsSent(NotificationHandler.SendNotification sende
         }
     }
 
+    private CcdNotificationWrapper buildGenericLetterBaseWrapper(Appellant appellant, NotificationEventType eventType, Representative representative,
+                                                                 List<CcdValue<DocumentSelectionDetails>> documentsSelection, boolean sscsDocument, boolean dwpDocument) {
+        var wrapper = buildBaseWrapper(appellant, eventType, representative, Benefit.PIP, "Online", READY_TO_LIST.getId());
+
+        if (documentsSelection != null) {
+            wrapper.getNewSscsCaseData().setDocumentSelection(documentsSelection);
+
+            var sscsLink = DocumentLink.builder().documentUrl("sscs_url").documentFilename("sscs").build();
+            var dwpLink = DocumentLink.builder().documentUrl("dwp_url").documentFilename("dwp").build();
+
+            if (sscsDocument) {
+                wrapper.getNewSscsCaseData().getSscsDocument().add(SscsDocument.builder()
+                        .value(SscsDocumentDetails.builder().documentLink(sscsLink).documentFileName("sscsDocument").build())
```

Comment:
`SscsDocumentDetails.builder().documentLink(sscsLink).documentFileName("sscsDocument").build()`
to separate var and reference the var